Source/GUI/MemWatcher/Dialogs/DlgAddWatchEntry.cpp

```diff
@@ -424,3 +437,23 @@ MemWatchEntry* DlgAddWatchEntry::stealEntry()
   m_entry = nullptr;
   return entry;
 }
+
+void DlgAddWatchEntry::onPointerOffsetContextMenuRequested(const QPoint& pos)
+{
+  QLabel* const lbl = static_cast<QLabel*>(sender());
+
+  QMenu* contextMenu = new QMenu(this);
+  QAction* copyAddr = new QAction(tr("&Copy Address"), this);
+
+  const QString text{QString::number(lbl->property("addr").toUInt(), 16).toUpper()};
```

Comment:
Question: Would we want to pad the string with heading 0s? (`14FA` vs `000014FA`)

Reply:
It wouldn't be necessary for MEM1. The "addr" property is set using the address value within the address space of Dolphin which has a base address of 0x80000000, so any offset into it should not need padding as it starts at 8 characters in hex. That being said, I'm not sure if MEM2 would need padding or not.

Reply:
Is there a blocker to merge this? @cristian64